lightning/src/ln/peer_handler.rs

```diff
@@ -980,6 +988,21 @@ impl<Descriptor: SocketDescriptor, CM: Deref> PeerManager<Descriptor, CM> where
 							}
 						}
 					},
+					MessageSendEvent::BroadcastNodeAnnouncement { ref msg } => {
```

Comment:
if Rust had a way of distributing match arms across multiple files, this would be the place

Reply:
I'm not sure what you mean here?

Reply:
I was referring to the fact that we're adding yet another match arm to a match that's hundreds of lines long.